SIPS/sip-63.md

```diff
@@ -92,7 +94,15 @@ interface ITradingIncentive {
 
 Remit exchange fees generated by an account to the `Trading incentive` contract, storing the total fees for each account in the contract.
 
-Referrals - A function to allow trading incentives to be recorded against any `address` instead of the `msg.sender` of the `synthetix.exchange()` event. The `address` will need to be the one claiming the rewards from the `trading incentive` contract.
+Referrals - A function to allow trading incentives to be recorded against any `address` instead of the `msg.sender` of the `synthetix.exchange()` transaction. The `address` will need to be the one claiming the rewards from the `trading incentive` contract.
+
+**Exchange Partner Volume Tracking**
+
+Partner exchanges such as `https://1inch.exchange/` who allow trading Synths on their frontend platform will be able to earn partner volume rewards that is paid separately by the Synthetix DAO.
+
+In order to support and integrate partner exchange volume tracking, the `Synthetix.exchange` function will be overrided to allow partner exchanges to pass in their unique tracking code (for the partner volume tracking) and also the `referrer` address that will be able to track the originating trader's individual address.
```

Comment:
"function will be overrided" did you mean "overloaded"?

Reply:
I believe he means to overload them with additional arguments - that way any existing implementations to `exchange()` will still work with the original signature. That being said, last time I looked ethers & web3 didn't love overloaded function names... it's not a huge deal but might it not be better to suffix the names of these functions to make them unique?